Jenkinsfile_CNP

```diff
@@ -107,20 +107,24 @@ withPipeline(type, product, component) {
       def aksServiceName = dockerImage.getAksServiceName().toLowerCase()
       def hrsStorageSecret = "storage-secret-hrs-${aksServiceName}"
       def cvpStorageSecret = "storage-secret-cvp-${aksServiceName}"
+      def hrsStorageSecretName = "storage-account-hrs-${aksServiceName}"
+      def cvpStorageSecretName = "storage-account-cvp-${aksServiceName}"
```

Comment:
I think we do not need `cvpStorageSecret` and `hrsStorageSecret`  ?

Reply:
We need hrsStorageSecret to build the ConnectionString. We need cvpStorageSecretName to build the CVP URL.